### PR TITLE
Add note about namespaced context classes with example

### PR DESCRIPTION
**Pull request by Bendihossan**
For example this is an issue where somebody may wish to namespace their classes to become PSR-2 compliant, but once namespaces behat can no longer find the contexts without this configuration.

### CODE REVIEW

File: guides/6.profiles.rst [hunk 1]
```diff
@@ -15,7 +15,21 @@ you want to autoload via ``behat.yml``:
     default:
         autoload:
             '': %paths.base%/app/features/bootstrap
-                
+
+If you wish to namespace your features (for example: to be PSR-2 complaint) you will need to add the namespace to your contexts and also add tell behat where to load them. Here ``contexts`` is an array of namespaced classes:
```
Comment (stof) [inline guides/6.profiles.rst]: it is an array of classes. The name of the class is `My\Application\Namespace\Bootstrap\FeatureContext` when you put the class in a namespace, not `FeatureContext`

**Comment by everzet** [comment]
And it's PSR-1, not PSR-2 AFAIR

**Comment by Bendihossan** [comment]
Thanks @stof and @everzet, I've amended that commit.

**Comment by Bendihossan** [comment]
@everzet Was something wrong with this change?

**Comment by everzet** [comment]
@Bendihossan no. Github closes PRs automatically when branch against which PR was sent is getting deleted. I deleted `master` in favour of versioned branches.

**Comment by everzet** [comment]
@Bendihossan sorry for that :( Can you send a PR again against the right branch?

**Comment by Bendihossan** [comment]
@everzet No problem! :)

**Pull request closed without merging**